documentation/operators/src/troubleshooting/vps-isp.md

```diff
@@ -49,3 +49,43 @@ To find the right IP configuration, contact your VPS provider for support to fin
 On self-hosted machine it's a bit more tricky. In that case as an operator you must be sure that your ISP allows for public IPv4 and IPv6 and then it may be a bit of playing around to find the right configuration. One way may be to bind your binary with the `--host` flag to local address `127.0.0.1` and run `echo "$(curl -4 https://ifconfig.me)"` to get a public address which you use to bond your Mix Node to `nym-api` via Nym wallet.
 
 It's up to you as a node operator to ensure that your public and private IPs match up properly.
+
+### Logs Pruning
```

Comment:
Small grammar thing, I'd flip this to 'Pruning Logs'

Reply:
- [x] done